app/scripts/lib/createRPCMethodTrackingMiddleware.test.js

```diff
@@ -416,6 +429,77 @@ describe('createRPCMethodTrackingMiddleware', () => {
       });
     });
 
+    it('should track Confirmation Redesign through ui_customizations prop if enabled', async () => {
+      const originalEnvConfirmationRedesign =
+        process.env.ENABLE_CONFIRMATION_REDESIGN;
+      process.env.ENABLE_CONFIRMATION_REDESIGN = true;
+
+      const req = {
+        method: MESSAGE_TYPE.PERSONAL_SIGN,
+        origin: 'some.dapp',
+      };
+      const res = {
+        error: null,
+      };
+      const { next, executeMiddlewareStack } = getNext();
+      const handler = createHandler();
+
+      await handler(req, res, next);
+      await executeMiddlewareStack();
+
+      expect(trackEvent).toHaveBeenCalledTimes(2);
+
+      expect(trackEvent.mock.calls[1][0]).toMatchObject({
+        category: 'inpage_provider',
```

Comment:
```suggestion
        category: MetaMetricsEventCategory.InpageProvider,
```

Reply:
thanks! 👍🏼  was using the same pattern in the test. updated the 10 instances in the file [223a699](https://github.com/MetaMask/metamask-extension/pull/24182/commits/223a6997243205354f41891e18dd936ba33d5226)